curp/src/rpc/mod.rs

```diff
@@ -84,15 +111,15 @@ impl FetchClusterResponse {
 
 impl ProposeRequest {
     /// Create a new `Propose` request
-    pub(crate) fn new<C: Command>(cmd: &C) -> bincode::Result<Self> {
-        Ok(Self {
-            command: bincode::serialize(cmd)?,
-        })
+    pub(crate) fn new<C: Command>(cmd: &C) -> Self {
```

Comment:
The `ProposeError` and `CurpError` also requires a From impl of bincode error, because some inter server communication still use bincode for serialization.